amp-camp/src/html/pages/index.html

```diff
@@ -14,10 +14,10 @@
     <main id="content" role="main" class="landing">
         <section class="hero-wrapper">
             <a href="/product-listing?gender=women&category=shirts" class="block">
-                <amp-img src="img/women-narrow.jpg" width="1.25" height="1" layout="responsive" media="(max-width: 52.06rem)" alt="Hero Landing 1" noloading="">
```

Comment:
we inherited these from the template. I assume you simply want to formalize these with HTML classes?